plugin/dnstap/setup.go

```diff
@@ -38,6 +39,26 @@ func parseConfig(c *caddy.Controller) (Dnstap, error) {
 	}
 
 	d.IncludeRawMessage = c.NextArg() && c.Val() == "full"
+	
+	d.Identity, _ = os.Hostname()
+	d.Version = caddy.AppName + "-" + caddy.AppVersion 
```

Comment:
Any reason to not use `coremain.CoreVersion` ? https://github.com/coredns/coredns/blob/45b0a11294c59bfd806a57807aaa2a185f761cd5/coremain/version.go#L5

Reply:
I'm happy to make that change. I just copied the version logic from the chaos plugin. https://github.com/coredns/coredns/blob/master/plugin/chaos/setup.go#L30
I much prefer this variable.

Reply:
On further investigation, it looks like coremain sets these values from those constants.
https://github.com/coredns/coredns/blob/master/coremain/run.go#L31
They're available as it is now without having to add an import. Any reason to not use these?

Reply:
I think it is okay to use these values to have consistency with chaos plugin